extensions/wikia/TemplateClassification/TemplateClassification.setup.php

```diff
@@ -101,3 +101,8 @@
 JSMessages::registerPackage( 'TemplateClassificationHints', [
 	'template-classification-bulk-classification-*'
 ] );
+
+JSMessages::registerPackage( 'TemplateClassificationGlobalShortcuts', [
+	'template-class-global-shortcuts-caption-classify-page',
+	'template-class-global-shortcuts-press-to-explore-shortcuts',
```

Comment:
Why is `global-shortcuts-press-to-explore-shortcuts` prefixed with `template-class-`? It doesn't seem specific to TemplateClassification and it's defined in the GlobalShortcuts extension.